### PR TITLE
update conferences

### PR DESCRIPTION
**Pull request by dougwoods3**

### CODE REVIEW

File: Events/index.md [hunk 1]
```diff
@@ -6,10 +6,14 @@ author: Doug, Jackson, & Tony
 
 # Upcoming Conferences
 * 17 February 2017 - Oregon State University [College of Engineering Graduate Research Showcase](http://engineering.oregonstate.edu/graduate-research-showcase-student-information-and-registration)
-  - Poster due 7 Feb
-* 11-15 June 2017 - American Nuclear Society Annual Meeting ([ANS](http://ansannual.org))
-  - Summary due 1 Feb
+  - Poster due 7 Feb.
+* 11-15 June 2017 - [2017 American Nuclear Society Annual Meeting](http://ansannual.org)
+  - Summary due 8 Feb.
```
Comment (albeanth) [inline Events/index.md:11]: haha aaaaaand it's pushed again

Comment (albeanth) [reply Events/index.md:11]: :shipit:

Comment (dougwoods3) [reply Events/index.md:11]: This makes it almost a month. @albeanth could have done some work and submitted by then 😆 .

Comment (albeanth) [reply Events/index.md:11]: hah! if it gets pushed another week maybe! 

(then again I've been saying that since the original deadline)

Comment (dougwoods3) [reply Events/index.md:11]: haha, i'm just giving you a hard time.

**Comment by harterj** [comment]
👍

**Pull request merged**